### PR TITLE
View role

### PR DESCRIPTION
**Pull request by dehou37**
## Pull Request Description

BF-03 View Role Details
<Please provide a brief description of the changes and the purpose of this pull request.>
Pull request to merge into main 
## Checklist (Definition of Done)

- [x] Unit tests passed
- [x] Code reviewed
- [x] Acceptance criteria met
- [x] Test cases passed
- [x] C4 diagram checked
- [x] Product owner accepted

## Screenshots (if applicable)
image

### CODE REVIEW

File: role-skill-match-app/resources/views/update-role.blade.php
Comment (joeyt4n) [inline role-skill-match-app/resources/views/update-role.blade.php:1]: Standardise 1 as part time, 2 as full time. Selected value is correct, if-else checking is wrong

File: role-skill-match-app/resources/views/view-role.blade.php
Comment (joeyt4n) [inline role-skill-match-app/resources/views/view-role.blade.php:1]: Standardise 1 as part time, 2 as full time. Selected value is correct, if-else checking is wrong

**Review by joeyt4n — CHANGES_REQUESTED** [review]

**Review by hx240 — CHANGES_REQUESTED** [review]
View role details changes:
- standardize 1 = part time, 2 = full time for view-roles

Navbar changes:
- staff/hr staff/manager dropdown button is not working or functional at all, please fix it
- "View Role Listings" on navbar should redirect to http://localhost:8000/browse-roles/ 
- There should not be 'Create Role Listing' / 'Edit Role Listing' menu items, please put a 'View Applications' placeholder nav-item for now (Can follow browse-roles branch for navbar guidance)

File: role-skill-match-app/resources/views/update-role.blade.php
Comment (joeyt4n) [reply role-skill-match-app/resources/views/update-role.blade.php:1]: Can also resolve conflict issue by accepting main

image

**Review by hx240 — APPROVED** [review]

**Review by joeyt4n — APPROVED** [review]

**Pull request merged**